dash/src/stores/useSettingsStore.ts

```diff
@@ -70,6 +73,9 @@ export const useSettingsStore = create<SettingsStore>()(
 				oledMode: false,
 				setOledMode: (oledMode: boolean) => set({ oledMode }),
 
+				useSafetyCarColors: false,
```

Comment:
I like this It can be a default on

```suggestion
				useSafetyCarColors: true,
```